spec/features/customers/show_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+RSpec.describe Customer, type: :feature do
+  before(:each) do
+    @supermarket = Supermarket.create(name: "Whole Foods", location: "124 Wall St.")
+    @customer = Customer.create(name: "John Dwayne")
+    @item1 = @customer.items.create(name: "Banana", price: 20)
```

Comment:
So you would want your item to be created like so 
```ruby
@item1 = @supermarket.items.create!(name: "Banana", price: 20)
```
Then you will also need to have a `CustomerItem` created as well like
```ruby
CustomerItem.create!(customer: @customer, item: @item1)
```